src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurRequestedCashAdvance.java

```diff
@@ -0,0 +1,86 @@
+package edu.cornell.kfs.concur.batch.businessobject;
+
+import java.sql.Date;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class ConcurRequestedCashAdvance extends PersistableBusinessObjectBase{
+    protected int concurRequestedCashAdvanceId;
```

Comment:
The maximum number that could be generated for this field with data types JDBC INTEGER and Java int are values with a minimum value of -2,147,483,648 and a maximum value of 2,147,483,647.  This table would only be dealing with the positive side of this range. With this table representing only cash advance payments that were requested and generated, I think we are are going to be ok with those two existing data types.